WIP-SQL/types-of-database-servers.md

```diff
@@ -0,0 +1,69 @@
+# Database server types
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+Depending on the user's needs, we can distinguish different types of database servers.
+
+*MySQL* is one of the world's most popular, open source databases. It is best known for its web application database services. It pays an important role in the so called LAMP[1] web development platform. MySQL supports many different platforms, such as: Microsoft Windows, Linux and Mac OS X.
+
+Some MySQL features:
+- high performance
+- open source freedom (cheapest option)
+- 24 x 7 support
```

Comment:
Change this to "a large community of professional users", 24/7 support is a product feature of paid users of MySQL offered by Oracle